testdata/src/go.uber.org/receivers/receivers.go

```diff
@@ -47,8 +47,30 @@ func (s *S) nonnilRecv() {
 	_ = s.f
 }
 
-func testCaller(dummy bool, i int) {
+func (s S) nonPointerRecv() {
```

Comment:
Same here, let's add cases where the receiver is the blank identifier :)

Reply:
Done :)